docs/fastdds/getting_started/simple_app/includes/task_publisher.rst

```diff
@@ -0,0 +1,157 @@
+    :language: C++
+    :lines: 35-59
+
+We will now explain the private fields of the class that will be necessary to understand in order to discuss the
+remaining public methods of the class.
+First, the ``hello_`` field is defined as an object of the HelloWorld class that defines the data type we created
+with the IDL file.
+Then the fields participant, publisher, topic and data writer are defined.
+Finally the ``type_`` object of the TypeSupport class is defined.
+This object will be used to register the topic data type in the DomainParticipant.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 134-146
+
+Finally, it is defined the PubListener class by inheriting from the DataWriterListener class, to override the data
+writer listener callbacks.
+These allow us to execute routines in case of an event.
+The overridden callback ``on_publication_matched`` allows you to define a series of actions when a new subscriber
+is detected listening to the topic the publisher is using.
+The ``info.current_count_change`` field allows us to detect these changes of subscribers that are matched to the
+publisher.
+This is a field in the MachedStatus structure that allows you to track changes in the status of subscriptions.
+Finally, the ``listener_`` object of the class is defined.
+
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 148-184
+
+Back to the public methods of the class, the next line defines the public publisher's initialization function.
+This method performs several actions:
+
+# Initializes the content of the HelloWorld type ``hello_`` structure fields.
+# Assigns a name to the participant through the QoS of the DomainParticipant.
+# Use the DomainParticipantFactory to create the participant.
+# Register the data type defined in the IDL.
+# Create the topic for the publications.
+# Create the publisher.
+# Creates the DataWriter with the listener previously created.
+
+As you can see, the QoS value for all entities, except for the participant's name, is the default value
+(``PARTICIPANT_QOS_DEFAULT``, ``PUBLISHER_QOS_DEFAULT``, ``SUBSCRIBER_QOS_DEFAULT``, ``TOPIC_QOS_DEFAULT``,
+``DATAWRITER_QOS_DEFAULT``, ``DATAREADER_QOS_DEFAULT``).
+The default value of the QoS of each DDS Entity can be checked in the
+`DDS standard <https://www.omg.org/spec/DDS/About-DDS/>`_.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 61-103
+
+To make the publication, the publish method is implemented. To make the publication, the publish method is implemented.
+This is actually a ``write`` of a change of the DataWriter object.
+
+.. literalinclude:: /../code/Examples/C++/DDSHelloWorld/src/HelloWorldPublisher.cpp
+    :language: C++
+    :lines: 105-115
+
+The public run method executes the action of publishing a given number of times.
```

Comment:
In general (extra anal warning): in c++, the correct naming is public/private member variable/function, never method, and never member alone